test/account/withdraw.ts

```diff
@@ -1,9 +1,10 @@
+// Declare imports
 import * as helpers from '../helpers/contract-helpers';
```

Comment:
This file dropped 6 lines of code, please just double check nothing was lost here